jdaviz/core/helpers.py

```diff
@@ -402,6 +407,85 @@ def show_in_new_tab(self, title=None):  # pragma: no cover
                       DeprecationWarning)
         return self.show(loc="sidecar:tab-after", title=title)
 
+    def get_data(self, data_label=None, cls=None, subset_to_apply=None, statistic=None):
+        """
+        Returns data with name equal to data_label of type cls with subsets applied from
+        subset_to_apply.
+
+        Parameters
+        ----------
+        data_label : str, optional
+            Provide a label to retrieve a specific data set from data_collection.
+        cls : `~specutils.Spectrum1D`, `~astropy.nddata.CCDData`, optional
+            The type that data will be returned as.
+        subset_to_apply : str, optional
+            Subset that is to be applied to data before it is returned.
+        statistic : {'minimum', 'maximum', 'mean', 'median', 'sum', 'percentile'}, optional
+            The statistic to use to collapse the dataset.
+
+        Returns
+        -------
+        data : cls
+            Data is returned as type cls with subsets applied.
+
+        """
+        list_of_valid_statistic_values = ['minimum', 'maximum', 'mean',
+                                          'median', 'sum', 'percentile']
```

Comment:
does percentile require passing another argument for the percentile itself?  (We don't seem to use this in cubeviz, so maybe we can just exclude from the list of supported statistics?)

Reply:
Good call, I checked cubeviz and it is not one of the options in the drop down.